### PR TITLE
[IMP] MRP: edited/updated Semi-Finished products doc v15

### PR DESCRIPTION
**Pull request by weka-odoo**
Replaced the images, and rewrote the content to ensure it's consistent and up to date.

### CODE REVIEW

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:33]:
```suggestion
Once the top-level product is configured, click the :guilabel:`Bill of Materials` smart button on
the product form and click :guilabel:`Create` to make a :abbr:`BoM (Bill of Materials)` for the
top-level product. Then, simply add the semi-finished products to this :abbr:`BoM (Bill of
Materials)`, along with any other necessary components.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:37]:
```suggestion
   :alt: Bill of materials for a top-level product, containing a subassembly component.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:55]:
```suggestion
**Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
the minimum and maximum desired stock quantities to `0`. To review how to set up reordering rules,
refer to :doc:`this page <../../../inventory_and_mrp/purchase/products/reordering>`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:60]:
```suggestion
Option 1 is more flexible than Option 2 and is therefore recommended. Reordering rules do not
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:67]:
```suggestion
Regardless of the method chosen, semi-finished products must be fully manufactured before
manufacturing can begin on the top-level product.
```

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hello @weka-odoo!

Thank you for your patience on this PR!

I found some edits for details and formatting. Please let me know if you have any questions. When you're ready, please tag me again for another look. Thank you :smile_cat:

**Comment by weka-odoo** [comment]
@mivu-odoo I incorporated the requested changes. Please take another look whenever you get the chance. Thank you!

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:21]:
```suggestion
   :alt: A bill of materials for a semi-finished product.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:28]:
```suggestion
--> Products --> Products`. Then, :guilabel:`Create` the top-level product. Configure the product’s
specifications as desired, and be sure to :guilabel:`Save`.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:37]:
```suggestion
   :alt: A bill of materials for a top-level product that contains a subassembly component.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (mivu-odoo) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:71]:
```suggestion
   :alt: A manufacturing order for a top-level product.
```

**Review by mivu-odoo — APPROVED** [review]
Hello @weka-odoo!

Thank you for your revisions! I found a few small edits for language.

Once you push those edits, feel free to go straight to ZST for final technical review, no need to tag me again. Thank you :smile_cat:

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:58]: Number of formatting suggestions here:
- Would first, format this as a bulleted list
- fix the apostrophe
- change '0' to `0` since it's input
- add `:guilabel:`'s to **Option 2** since these are checkboxes on the UI and they are being directly addressed to activate.

As well the `To review how to...` sentence might serve better as a `.. :seealso:` here, just like the one in the previous comment. Another option is to place an accumulated list of related documentation articles either at the beginning of the doc so the reader gets a heads up to be comfortable with those topics or at least have them open in a new tab. Up to you 🙃 

```suggestion
- **Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
  the minimum and maximum desired stock quantities to `0`.

  .. seealso::
     - :doc:`Don't run out of stock with reordering rules
       <../../../inventory_and_mrp/purchase/products/reordering>`

- **Option 2:** Activate the :guilabel:`Replenish on Order (MTO)` and :guilabel:`Manufacture`
  routes under the :guilabel:`Inventory` tab of the semi-finished product's product form.
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:62]: Suggestion since we're _referencing_ something on the UI but not in an instructional way. 
Feel free to ignore if you disagree.

```suggestion
to other orders, if necessary. The *Replenish on Order (MTO)* route creates a unique link between the
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:69]: Replace underscores with hyphens.
As well, [media file names can not have capital letters](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files).
```suggestion
.. image:: sub_assemblies/semifinished-on-mo.png
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hey @weka-odoo 

This was a great rewrite!

I have a few suggestions around the following:
- serif vs. sans-serif apostrophes
- media file names
- tag and list formatting

I like what you did to try and help the reader by adding in references to other articles. I offered a couple suggestions there with some options.

Let me know if you have any q's otherwise I'll look forward to the revisions, thanks!

cc: @mivu-odoo

**Comment by StraubCreative** [comment]
@weka-odoo quick follow up— I had to solve a merge conflict that showed up as a result of #2983 which wiped all trailing whitespaces across the entire repo, including this file. You should be all set on [220e1c8](https://github.com/odoo/documentation/pull/2787/commits/220e1c8c92534db01bb75fe8df30ea369638a5ed), please make sure you `git pull --rebase` before you start addressing the CR's 👍

**Review by StraubCreative — APPROVED** [review]
All change requests addressed in [0b0adb4](https://github.com/odoo/documentation/pull/2787/commits/0b0adb4231d138600655cfcf75d40237d9dad77d). Moving on to DR.

**Comment by StraubCreative** [comment]
Hi @odoo/doc-review 👋 
This article about sub-assemblies is ready for your review. 
Thank you 🙂

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:58]:
```suggestion
  .. seealso::
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:60]: No need to put a single seealso ref in a bulleted list.
```suggestion
     :doc:`../../purchase/products/reordering`
```

File: content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst
Comment (AntoineVDV) [inline content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst:19]:
```suggestion
   :doc:`bill_configuration`
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by StraubCreative** [comment]
@robodoo r+

**Comment by StraubCreative** [comment]
@AntoineVDV should be good to go unless you catch anything else.
Please advise weka-odoo will not be able to FP the chain once the merge command is issued. 
Will need you to do it please or can delegate to me or @mivu-odoo if that's possible 🙂

**Comment by AntoineVDV** [comment]
Hi @StraubCreative, [the r+ is already delegated to you](https://github.com/odoo/documentation/pull/2787#pullrequestreview-1233636996), and so will the forward-port PRs. ;)

Note that the forward-port bot is currently broken (I hope for a fix next week :crossed_fingers:). It will create the forward-port PRs, but it will also rebase them on a commit from December 2... This highly increases the chances of having a "conflict". If that happens in the fwp PR for saas-15.2, run `git fetch origin saas-15.2 <the-fwp-branch> && git switch <the-fwp-branch> && git reset --hard origin/saas-15.2 && git cherry-pick ea43afc792b94de938c0f0cb8546a6a22675a7f1 && git push --force-with-lease` to fix it.

**Comment by StraubCreative** [comment]
@robodoo r+

**Comment by AntoineVDV** [comment]
@StraubCreative My bad; I thought you were the author of the PR.

@robodoo r+

**Pull request closed without merging**
